app/services/search/query_builders/feed_content.rb

```diff
@@ -37,6 +37,21 @@ class FeedContent < QueryBase
         body_text
       ].freeze
 
+      SOURCE = %i[
```

Comment:
In the future, I foresee these being configurable but for now I am hardcoding them in the class as defaults.